### DISCUSSION TITLE
Move to Apple Silicon breaks my GIA

### DISCUSSION THREAD

**Discussion by badgerbreath** [Ascoware/get-iplayer-automator#477] [Q&A] [answered]
Big fan of GIA which I have used gratefully for years. However I just migrated to a a New M3 MacBook Pro from an Intel MacBook Pro. The Intel and the M3 were both on Sonoma 14.4.1. When I launch GIA on the M3 it seems stuck 'Updating Programme Index Feeds' which has been running for a few hours to no avail.

**Comment by skovatch** [comment, marked as answer]
Try this: go to Get iPlayer Automator.app in the Finder, and choose Get Info. Look for 'Open using Rosetta' and check that.

Also, what version do you have? Make sure you're using 1.26.1 first -- 1.26.0 had issues with a missing symlink that kept get_iplayer from running.

**Reply by badgerbreath** [reply]
That works! Many thanks, I didn't realise that Rosetta was still available for Apple Silicon and when the program launched without an error I thought it must be some other issue. Good work very much appreciated and sorry to take up your time.

**Reply by skovatch** [reply]
No worries, happy to help. You’re not the first person to encounter this. I’m wondering if it has something to do with migrating from Intel to Apple Silicon.

**Reply by hugofalkman** [reply]
@skovatch  You may want to add this to the release info.

Some universal apps (apps that don't need Rosetta) include the setting “Open using Rosetta.” This setting enables a universal app such as a web browser to use plug-ins, extensions, or other add-ons that do need Rosetta, because they haven't been updated to support Apple silicon. If a universal app doesn't recognize an add-on that you installed for the app, you can quit the app, select this setting, and try again. (From https://support.apple.com/en-us/102527 )
